### ISSUE TITLE
Use ⌘1-9 to switch tabs, maybe ⌃1-2 or ⇧⌃1-2 for commit/history view

### ISSUE THREAD

**Issue by elia** [ForkIssues/Tracker#671]
Although I recognize that most of the keyboard shortcuts are consistent to those of GitX, most mac apps use ⌘1-9 to switch tabs, please consider alternatives for switching commit/history.

Possible alternatives 
- ⌃1-2
- ⇧⌃1-2
- ⌃⌥1-2
- ⌥⌘C (as a toggle)

_But I have no clear winner for that_

**Comment by DanPristupov** [comment]
This is slightly an off-topic, but I'd like to mention that Fork has 2 pairs of shortcuts to switch tabs:
Show Previous tab: ⌘⇧[ or ⌃⇧⇥
Show Next tab: ⌘⇧[ or ⌘⇧]

**Comment by elia** [comment]
Yeah thanks, I use them! And was really happy those were there 👍 🙂

**Comment by stk-ableton** [comment]
Hm, I switch between commit/history all the time, and I'm used to ⌘1/⌘2 for that; I would hate to see these go away, or change to something that requires more modifiers.

**Comment by kuthair** [comment]
Hi, I too would love to see ⌘1-9 to switch tabs like in Safari and other apps. Also, now in PyCharm, yay!

Thanks btw, Fork is hands down best Git app.
